File: logica-programacao/javascript/033.js
/* Receba um número.
Mostre True se ele NÃO estiver entre 1 e 100.
 */

let numero = prompt("Número: ");
let intNumero = parseInt(numero);

let verificacao = !(intNumero >= 1 && intNumero <= 100);

let stringVerificacao = `O número NÃO está entre 1 e 100? ${verificacao}`;

console.log(stringVerificacao)

File: logica-programacao/python/033.py
# Receba um número.
# Mostre True se ele NÃO estiver entre 1 e 100.

numero = input("Número: ")
int_numero = int(numero)

verificacao = not (int_numero >= 1 and int_numero <= 100) 

string_verificacao = f"O número NÃO está entre 1 e 100? {verificacao}"

print(string_verificacao)